examples/simple-py/__main__.py

```diff
@@ -1,7 +1,7 @@
 import json
 import pulumi
 import pulumi_aws as aws
-import pulumi_apigateway as apigateway
+import pulumi_aws_apigateway as apigateway
```

Comment:
Any comment on this change?

Reply:
This test was never run .. so I fixed it 😬